docs/e2e.md

```diff
@@ -0,0 +1,13 @@
+# End-to-end testing with Cypress
+
+## Run Locally
+
+* `npm run build`
+* `cypress:open` Open Cypress task runner
+* `cypress:cli` Run Cypress on the terminal
+* Or additionally, you can use the command `cypress:travis` to run with Travis
+
+## Write tests
+
+* Create a file under ./cypress/integration/
```

Comment:
should be using `

- Create a file under `./cypress/integration/`

I think it also needs some info on the naming convention for the file names?